packages/runtime/container-runtime/src/dataStore.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export interface IDataStoreAliasMessage {
```

Comment:
Can you please add /** */ comments above each interface, and possibly - for each field?